api/config.js

```diff
@@ -9,7 +9,9 @@ const PRODUCTION_IDENTIFIER = 'production';
 const TEST_IDENTIFIER = 'test';
 
 const config = {};
-config.auth = {};
+config.auth = {
+  githubclient: {}
```

Comment:
Not something that needs to change, but I feel like this is a little verbose. Is the key `config.auth.github` being used elsewhere?